web/modules/custom/demo_content/demo_content.install

```diff
@@ -232,18 +232,6 @@ function demo_content_get_asset_data() {
 <section id="status"><p>&nbsp;</p><p><span style="font-size: 13px">Copyright &copy; 2012 European Union. This vocabulary is published under the&nbsp;</span><span style="font-size: 13px"><a href="https://joinup.ec.europa.eu/category/licence/isa-open-metadata-licence-v11" rel="license" style="font-size: 13px">ISA Open Metadata Licence v1.1</a></span></p>
 </section>',
       // These lines are commented because the entities do not currently exist.
-      // 'field_is_distribution' => [
```

Comment:
Why are these removed? these were intended to be uncommented when the taxonomy terms existed.